uber/configspec.ini

```diff
@@ -1068,13 +1068,14 @@ __many__ = string
 
 [[badge]]
 attendee_badge  = string(default="Attendee")
-supporter_badge = string(default="Supporter")
 staff_badge     = string(default="Staff")
+contractor_badge = string(default="Contractor")
 guest_badge     = string(default="Guest")
 one_day_badge   = string(default="One Day")
 __many__ = string
 
 [[badge_status]]
+pending_status = string(default="Pending")
```

Comment:
This is an invalid badge status that represents badges that have been created by admins but not approved of yet.